adoc/i-Introduction/SA.adoc

```diff
@@ -1,40 +1,42 @@
+- improve, simplify and unify the installation of SAP Landscape on SUSE Linux Enterprise Server for SAP Applications 
+- clearly standardize deployments and allow customers to use one set of tooling for multiple use cases such as Command Line interface, a GUI driven process, SUSE Manager or other automation framework.
 
-So it was clear for us to move to a more modern approach, like infrastructure-as-code, in order to help our customers to reduce the effort and errors.
+To achieve this, SUSE have adopted a more modern approach, infrastructure-as-code which helps customers reduce the effort and errors during deployments.
 
-As SUSE Linux Enterprise Server and many other SUSE products ship with a universal configuration management solution since a few years, we used it as the base for the new automation. 
+In recent years, SUSE Linux Enterprise Server and many other SUSE products shipped with a universal configuration management solution, this is used as the foundation for the new automation capability. 
 
-This configuration infrastructure management system is called Salt from SaltStack and provides a highly scalable, powerful and fast infrastructure automation and management, built on a dynamic communication bus.
-Salt can be used for data-driven orchestration, remote execution for any infrastructure, configuration management for any app stack, and much more.
+This configuration management system is called Salt from SaltStack and provides a highly scalable, powerful and fast infrastructure automation and management, built on a dynamic communication bus.
+Salt can be used for data-driven orchestration, remote execution for any infrastructure, configuration management for any app stack.
 
-Combining this management system with a infrastructure deployment solution like Terraform from Hashicorp makes it possible to do a hands free setup of an SAP Landscape, ready to login to start customizing your SAP System.
+Combining this configuration management system with an infrastructure deployment solution such as Terraform from Hashicorp makes it possible to do a hands free, error free setup of an SAP Landscape.  Once the deployment is complete, administrators can login to start customizing the SAP System.
 
-As last step, we added the possibility to allow insights into your SAP Landscape with comprehensive dashboards, realtime and historic views and active alerts and reporting based on flexible and powerful open-source projects Prometheus and Grafana.
+#As part of the deployment, SUSE added the ability provide insights into your SAP Landscape with comprehensive dashboards, realtime and historic views and active alerts and reporting based on flexible and powerful open-source projects Prometheus and Grafana.
 
-The deployment automation can be configured to also setup a monitoring environment for the clusters, HANA and Netweaver.
+The deployment automation can also be configured to setup a monitoring environment for the SAP HANA and SAP NetWeaver clusters.
```

Comment:
i guess we do not need this part

Reply:
I think i was duplicated later on, so I removed it.